include/circt/Dialect/FIRRTL/FIRRTLOpInterfaces.td

```diff
@@ -0,0 +1,39 @@
+include "mlir/IR/OpBase.td"
+
+def FModuleLike : OpInterface<"FModuleLike"> {
+  let cppNamespace = "circt::firrtl";
+  let description = [{ Provide common  module information. }];
+  let methods = [
+    InterfaceMethod<[{Get the module name} }], 
```

Comment:
FYI you can use `"Get the module name"` instead of [{..}] syntax.  The [{ syntax is intended for multi-line things.